shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -517,8 +518,14 @@ private CharSequence getClipboardData(PlatformChannel.ClipboardContentFormat for
       if (clip == null) return null;
       if (format == null || format == PlatformChannel.ClipboardContentFormat.PLAIN_TEXT) {
         ClipData.Item item = clip.getItemAt(0);
+        AssetFileDescriptor assetFileDescriptor;
         if (item.getUri() != null)
-          activity.getContentResolver().openTypedAssetFileDescriptor(item.getUri(), "text/*", null);
+          assetFileDescriptor =
+              activity
+                  .getContentResolver()
+                  .openTypedAssetFileDescriptor(item.getUri(), "text/*", null);
+        CharSequence charSequence = item.coerceToText(activity);
+        if (assetFileDescriptor != null) assetFileDescriptor.close();
```

Comment:
The other linter suggestions were straightforward to resolve, but I'm not sure if this is the right fix for this one.
The linter error was 
```
../../../flutter/shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java:521: Error: This AssetFileDescriptor should be freed up after use with #close() [Recycle]
          activity.getContentResolver().openTypedAssetFileDescriptor(item.getUri(), "text/*", null);
```
I can't actually tell what the AssetFileDescriptor is doing here, unless its creation somehow interacts with [`ClipData.Item.coerceToText`](https://developer.android.com/reference/android/content/ClipData.Item#coerceToText(android.content.Context))? I just decided to close it after getting the `CharSequence`, but I'm not sure if thats right.

Reply:
I'm moving this to a new PR